main/EDPftvarcon.F90

```diff
@@ -48,9 +48,11 @@ module EDPftvarcon
      real(r8), allocatable :: crown(:)               ! fraction of the height of the plant 
                                                      ! that is occupied by crown. For fire model. 
      real(r8), allocatable :: bark_scaler(:)         ! scaler from dbh to bark thickness. For fire model.
-     real(r8), allocatable :: crown_kill(:)          ! scaler on fire death. For fire model. 
+     real(r8), allocatable :: crown_resist(:)        ! resistance to fire. (1 = none) For fire model.
```

Comment:
I feel this naming convention is un-intuitive.  If the value is increasing, one would think the resistance is increasing, but its the opposite.

Reply:
yep, good point. rolling back to crown_kill and updated comments